### ISSUE TITLE
Guitar Hero black screen

### ISSUE THREAD

**Issue by pulsed3x** [piratesephiroth/TeconmoonWiiVCInjector#4]
Every time I load any Guitar Hero game (made in or after 2009), it gives me a black screen. Any fix?

**Comment by E1ite007** [comment]
Yeah, some Guitar Hero games even after patching slot 249 with base 56 don't work. Afak, these are Guitar Hero 5 and Guitar Hero: Warrior's of Rock.
This isn't an error on Teconmoon's tool neither whichever fork you try to use.
